.storybook/stories/templates/CaseDetail-page.stories.mdx

```diff
@@ -242,10 +243,34 @@ export const copyright = "© 2022 Gemeente Den Haag";
         </Sidenav>
         <main className="denhaag-page-content__main">
           <Heading2>Aanvraag subsidie geluidsisolatie</Heading2>
+          <Action
+            className="denhaag-page-content__task"
+            deadline={new Date().setDate(new Date().getDate() + 3)}
+            actions={
+              <a href="#example" class="denhaag-button">
```

Comment:
Not really important, but we probably want to use the Den Haag Button component, instead of the html anchor tag.